spec/url_validator_spec.rb

```diff
@@ -41,18 +63,123 @@ class Post
     ]
 
     valid_urls.each do |url|
-      it "accept valid url: #{url}" do
+      it 'accept valid url: #{url}' do
```

Comment:
Running the test with `bundle exec rspec --format doc` reveals the literal `#{` symbols in the test names.